community/contributing/releasing/index.md

```diff
@@ -22,10 +22,12 @@ include all commits merged into `master` for each service. We do this on a
 In order to be considered safe to merge into a release branch, your patch must:
 
 * Fix a documented regression. This means that the currently broken
-  functionality must have worked as expected in a previous version of Spinnaker.
-  If the regression is not already documented in a GitHub issue, please create
-  one. Describe the difference between the expected and observed behavior, and
-  include links to the commit(s) that introduced the regression.
+  functionality must have worked as expected in the version(s) of Spinnaker
```

Comment:
Does the version need to be one that is still supported and if so how can a user figure out which versions are still supported? Sorry if this is a silly question.

Reply:
No question is a silly questions here!
A user can view it on the releases page: https://spinnaker.io/community/releases/versions/
<kbd>![image](https://user-images.githubusercontent.com/3469532/88580337-50a83600-d000-11ea-8e04-cfc0a03c372a.png)</kbd>

or from halyard by doing a `hal version list`
<kbd>![image](https://user-images.githubusercontent.com/3469532/88580490-82b99800-d000-11ea-8a8d-baefc92f2b30.png)</kbd>

Reply:
👍  Updated to include link to supported versions!